### ISSUE TITLE
ui5-button: aria-expanded not working for screen reader

### ISSUE THREAD

**Issue by SherrySherry123** [SAP/ui5-webcomponents#1705]
i am from Successfactors team, just find the aria-expanded attribute does not work for screen reader;

**Describe the bug**
ui5-button with **aria-expanded=true/false** is not read out by screen reader;

**To reproduce**
1. set **aria-expanded=true** for ui5-button
`<ui5-button aria-expanded="true">Enabled</ui5-button>`
2.While using any screen reader (JAWS, NVDA), tab over the button that has aria-expanded=true
3."expanded/collapsed" never be read out

**Expected behavior**
when **aria-expanded=true**, a screen reader will announce "Expanded"
when **aria-expanded=false**, a screen reader will announce "Collapsed"

https://sap.github.io/ui5-webcomponents/playground/components/Button/
